### PR TITLE
Select ImGui explorer

### PR DESCRIPTION
**Pull request by DamianMadden**
Makes clicks onto scene item imgui elements select the item
Resolves [AB#544](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/544)

### CODE REVIEW

File: src/scene/vcFolder.cpp [hunk 1]
```diff
@@ -342,7 +342,12 @@ void vcFolder::HandleImGui(vcState *pProgramState, size_t *pItemID)
         ImGui::Indent();
         ImGui::PushID(udTempStr("SXIExpanded%zu", *pItemID));
 
+        ImGui::BeginGroup();
         pSceneItem->HandleImGui(pProgramState, pItemID);
+        ImGui::EndGroup();
+
+        if (ImGui::IsItemClicked(0))
+          vcProject_SelectItem(pProgramState, m_pNode, pNode);
```
Comment (OsiJr) [inline src/scene/vcFolder.cpp]: I would probably recommend not doing it this way.

The majority the UI does not need to do the selection and this behaviour is not consistent with how other applications behave. The bug can be better fixed by showing the elements when the selected item is `-1`. Changing the speed of an attached model shouldn't select the item for example.

Comment (DamianMadden) [reply src/scene/vcFolder.cpp]: So, you're suggesting just closing the PR and the AB item?

Comment (DamianMadden) [reply src/scene/vcFolder.cpp]: or you want selection to open the treenode for an items handleimgui?

**Comment by OsiJr** [comment]
Going to close this and close the issue as well.

**Pull request closed without merging**